{
  "parameter(s)": [
    -56
  ],
  "test_sublist_size": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00017
  ],
  "test_list_of_several_same_number": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00036
  ],
  "test_list_of_single_number": [
    "failed",
    "TypeError: 'int' object is not callable",
    0.00019
  ]
}